tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,2 @@
-<h2 id="title">Welcome to Ember</h2>
-
+{{#tf-toaster}}{{/tf-toaster}}
```

Comment:
Since you're not yielding anything, it's probably fine to just make this inline `{{tf-toaster}}`